documentation/data/link-type.md

```diff
@@ -5,7 +5,16 @@ uid: linkType
 # Link Data Messages
 
 
-A Link is used to create relationships between Types and instances of Types, and is defined using the pre-defined typeid `__Link` in the Data Message.
+A Link is used to create relationships between omf created resources.  Links are defined using the pre-defined typeid `__Link` in the Data Message. 
```

Comment:
Extremely minor - we might capitalizing OMF.

Reply:
Done